dbt_copilot_helper/commands/dns.py

```diff
@@ -526,11 +526,24 @@ def get_load_balancer_domain_and_configuration(
     with open(f"./copilot/{svc}/manifest.yml", "r") as fd:
         conf = yaml.safe_load(fd)
         if "environments" in conf:
-            for domain in conf["environments"].items():
-                if domain[0] == env:
-                    domain_name = domain[1]["http"]["alias"]
-
-        # What happens if domain_name isn't set? Should we raise an error? Return default ? Or None?
+            if env in conf["environments"]:
+                for domain in conf["environments"].items():
+                    if domain[0] == env:
+                        if (
+                            domain[1] is None
+                            or domain[1]["http"] is None
+                            or domain[1]["http"]["alias"] is None
+                        ):
+                            click.secho(
+                                "No domains found, please check the manifest file", fg="red"
```

Comment:
Check which manifest file? Might be useful to show the path here, just for better UX.

Reply:
sure, will add path.